### PR TITLE
Added ADMINS to settings.py

### LINKED ISSUE #49: Add Admins to settings.py

### PR DESCRIPTION
**Pull request by Utsal20**
Fixes #49 completed. Admins are added to settings.py

### CODE REVIEW

**Review by nikhita — APPROVED** [review]

**Comment by nikhita** [comment]
@Utsal20 Thanks! I'll let @jarifibrahim take a look at it too before merging it. :+1:

**Comment by tapaswenipathak** [comment]
@Utsal20 Can you please add @amarlearning (amar.om1994@gmail.com) too?  and then this can be merged.

**Comment by nikhita** [comment]
@Utsal20 I modified the PR description to read `Fixes #49` instead of `Issue #49`. When we write like this, it automatically closes the issue once this PR is merged! :smile: 

More details here: https://github.com/blog/1506-closing-issues-via-pull-requests, https://help.github.com/articles/closing-issues-via-commit-messages/.

**Review by jarifibrahim — APPROVED** [review]
LGTM. I'll merge after you add Amar Prakash Pandey (amar.om1994@gmail.com) to Admins.

File: oshc/oshc/settings.py
Comment (amarlearning) [inline oshc/oshc/settings.py]: I think we should split the list something like this for better readability. [[more read](http://pep8.org/#maximum-line-length)]
```python
ADMINS = [
    ('Tapasweni Pathak','tapaswenipathak@gmail.com'),
    ('Nikhita Raghunath','nikitaraghunath@gmail.com'),
    ('Ibrahim Jarif','jarifibrahim@gmail.com')
]
```
Extraneous change. Having two consecutive new lines is not a good practice. [[more read](http://pep8.org/#blank-lines)]

**Review by amarlearning — CHANGES_REQUESTED** [review]

File: oshc/oshc/settings.py
Comment (Utsal20) [reply oshc/oshc/settings.py]: Thank you for your suggestions!
I just did:
git add oshc/oshc/settings.py
git push -u origin master
Will this be enough? Or is there something I am missing?

**Comment by tapaswenipathak** [comment]
Can I please merge this one @amarlearning? or you would suggest @Utsal20 to squash commits? 😄

I see it's @Utsal20's first pull request. ✨

**Comment by tapaswenipathak** [comment]
I am merging this sorry, can't wait. Congrats for your first pull request @Utsal20! 🎆 

But @Utsal20 there is something as [squashing commits](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) in which if you have fixed your previous commit or you have many related commits separate you can squash them as one. I am sure these two would have suggested you to do this before merging, so surely have a read, for all next pull requests you have to. :)

**Comment by tapasweni-pathak** [comment]
I am merging this sorry, can't wait. Congrats for your first pull request @Utsal20! 🎆 

But @Utsal20 there is something as [squashing commits](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) in which if you have fixed your previous commit or you have many related commits separate you can squash them as one. I am sure these two would have suggested you to do this before merging, so surely have a read, for all next pull requests you have to. :)

**Comment by Utsal20** [comment]
Thank you so much for your support guys! Will do the reading.

**Pull request merged**
